cmd/up.go

```diff
@@ -138,7 +138,25 @@ func Up() *cobra.Command {
 
 			dev, err := utils.LoadDev(devPath)
 			if err != nil {
-				return err
+				if !strings.Contains(err.Error(), "okteto init") {
+					return err
+				}
+				if !utils.AskIfOktetoInit(devPath) {
+					return err
+				}
+
+				workDir, err := os.Getwd()
+				if err != nil {
+					return err
```

Comment:
in this ase don't send the raw error, but instead something that explains what happened.